README.md

```diff
@@ -107,6 +107,8 @@ set OPENSSL_LIBS=libcrypto64MT:libssl64MT
 
 `$ cargo install sccache`
 
+<sub>_use `--features=all` for non-local caching, see [build requirements](#build-requirements)_</sub>
```

Comment:
This is actually not completely correct since `S3` is in the list of [default features](https://github.com/mozilla/sccache/blob/5112041e0b1f95467e1a4da758eb2533bfb00d58/Cargo.toml#L119) and `non-local`.
The list of optional features in the `Build` section is outdated - think `memcached` is missing here - and this PR cries for adding that ;-)